test/fixup-svg-string.js

```diff
@@ -27,3 +27,20 @@ test('fixupSvgString should make parsing fixtures not throw', t => {
     });
     t.end();
 });
+
+test('fixupSvgString should correct invalid mime type', t => {
+    const filePath = path.resolve(__dirname, './fixtures/invalid-cloud.svg');
+    const svgString = fs.readFileSync(filePath)
```

Comment:
`fs.readFileSync(file, 'utf8')` will let you skip the `.toString()` on the buffer - passing the encoding to node file API returns strings instead of buffers